MorphicWin/QuickStrip.xaml.cs

```diff
@@ -301,6 +302,7 @@ public List<Item> Items
 
         private void Zoom(object sender, QuickStripSegmentedButtonControl.ActionEventArgs e)
         {
+            Countly.RecordEvent("change-zoom");
```

Comment:
I went with events that were more basic, like "change-zoom" instead of "zoom-in" and "zoom-out" because I'm not... sure what type of useful data it would be to know that they zoom in more than they zoom out.